code/modules/mob/living/simple_animal/hostile/gingerbreadmobs.dm

```diff
@@ -0,0 +1,392 @@
+	mob_property_flags = MOB_CONSTRUCT
+	min_oxy = 0
+	max_oxy = 0
+	min_tox = 0
+	max_tox = 0
+	min_co2 = 0
+	max_co2 = 0
+	min_n2 = 0
+	max_n2 = 0
+	minbodytemp = 0
+	speed = 5
+
+/mob/living/simple_animal/hostile/gingerbomination
+	name = "gingerbomination man"
+	desc = "Stale dough and frosting holds loose, pale heaps of flesh together. The tiniest spark of the man he once was still trembles in his eyes. Although not entirely sentient he is at the very least aware and awake. This poor creature has a pretty crumby life"
+	icon_state = "gingerbominationman"
+	icon_living = "gingerbominationman"
+	icon_dead = "gingerbominationman_dead"
+	environment_smash_flags = 0
+	move_to_delay = 2
+	turns_per_move = 20
+	speak_chance = 1
+	speak = list("h-hhhe-el", "wh-why")
+	speak_emote = list("crumbles","sputters","whimpers")
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/human
+	//Why did I do this. This started as a cute gingerbread house.
+	response_help = "doughs up"
+	response_disarm = "crumbs"
+	response_harm = "breaks into"
+	maxHealth = 30
+	health = 30
+	attacktext = "stabs"
+	melee_damage_lower = 10
+	melee_damage_upper = 15
+	attack_sound = "sound/weapons/pierce.ogg"
+	vision_range = 8
+	aggro_vision_range = 8
+	idle_vision_range = 8
+	mob_property_flags = MOB_CONSTRUCT
+	min_oxy = 0
+	max_oxy = 0
+	min_tox = 0
+	max_tox = 0
+	min_co2 = 0
+	max_co2 = 0
+	min_n2 = 0
+	max_n2 = 0
+	minbodytemp = 0
+	speed = 3
+
+/mob/living/simple_animal/hostile/gingerbroodmother
+	name = "gingerbroodmother"
+	icon = 'icons/mob/giantmobs.dmi'
+	icon_state = "gingerbroodmother"
+	icon_living = "gingerbroodmother"
+	icon_dead = "gingerbroodmother_dead"
+	pixel_x = -16 * PIXEL_MULTIPLIER
+	wander = 0
+	move_to_delay = 40
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/gingerbroodmother
+	meat_amount = 1
+	ranged = 1
+	vision_range = 8
+	aggro_vision_range = 9
+	idle_vision_range = 8
+	speed = 10
+	maxHealth = 400
+	health = 400
+	harm_intent_damage = 5
+	melee_damage_lower = 20
+	melee_damage_upper = 35
+	attacktext = "crumbles"
+	ranged_cooldown = 2
+	ranged_cooldown_cap = 18
+	mob_property_flags = MOB_NO_LAZ
+	//This dough's hard baked
+	min_oxy = 0
+	max_oxy = 0
+	min_tox = 0
+	max_tox = 0
+	min_co2 = 0
+	max_co2 = 0
+	min_n2 = 0
+	max_n2 = 0
+	minbodytemp = 0
+	speed = 5
+
+/mob/living/simple_animal/hostile/gingerbroodmother/OpenFire(var/the_target)
+	var/mob/living/simple_animal/hostile/gingerbomination/A = getFromPool(/mob/living/simple_animal/hostile/gingerbomination,src.loc)
+	A.GiveTarget(target)
+	A.friends = friends
+	A.faction = faction
+	return
+
+/mob/living/simple_animal/hostile/gingerbroodmother/AttackingTarget()
+	OpenFire()
+
+/mob/living/simple_animal/hostile/gingerbroodmother/death(var/gibbed = FALSE)
+	..(gibbed)
+	visible_message("What could be described as flesh crumbles apart and slough off the doughy monstrosity. In an instant her once wriggling abdomen becomes still and she crumbles to the floor")
```

Comment:
"In an instant, her once wriggling abdomen becomes still and she crumbles to the floor."